src/Simplex/Messaging/Agent/Store/SQLite/Migrations/M20240124_file_redirect.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Messaging.Agent.Store.SQLite.Migrations.M20240124_file_redirect where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20240124_file_redirect :: Query
+m20240124_file_redirect =
+    [sql|
+ALTER TABLE snd_files ADD COLUMN redirect_size INTEGER;
+ALTER TABLE snd_files ADD COLUMN redirect_digest BLOB;
+
+ALTER TABLE rcv_files ADD COLUMN redirect_id INTEGER REFERENCES rcv_files ON DELETE CASCADE;
```

Comment:
Are temporary and unencrypted redirect descriptions files deleted?
There are two cases:
- on cleanup - they may be deleted if are read from db first
- on delete api call
It may be better to explicitly check in both cases if file has a redirect file and delete in this case

Reply:
Also redirect file can be deleted after destination rcv file has been updated

Reply:
Added received yaml deletion right after reading it.

Redirect records should be picked up by agent's clean up manager just like the ordinary files.

Reply:
Redirect records will be transitively deleted in db due to ON DELETE CASCADE and will not be picked up by cleanup manager for file deletion.

For delete api record of the file itself is always deleted first and redirect record deleted transitively, so the problem is even more acute than with cleanup manager where order may differ.